modules/oauth-web-worker/src/oauth-worker.ts

```diff
@@ -48,7 +42,7 @@ import {
 } from "./models";
 import { getCodeChallenge, getCodeVerifier, getJWKForTheIdToken, isValidIdToken } from "./utils";
 
-export const OAuthWorker: OAuthWorkerSingletonInterface = (function (): OAuthWorkerSingletonInterface {
+export const OAuthWorker: OAuthWorkerSingletonInterface = (function(): OAuthWorkerSingletonInterface {
```

Comment:
Is there a special reason for not having an arrow function here?

Reply:
This is a self-calling function and is needed for closure.